src/auth/auth.service.ts

```diff
@@ -0,0 +1,73 @@
+import { HttpException, HttpStatus, Injectable } from '@nestjs/common';
+import { JwtService } from '@nestjs/jwt';
+import { ApiMessages } from 'src/common/constants';
+import { Env } from 'src/env';
```

Comment:
PullProofAI

The `Env` import seems to be missing in the new code hunk, which could lead to potential runtime errors due to undefined constants.

```diff
- import { Env } from 'src/env';
+ import { Env } from '../env'; // Assuming 'env' is at the same level as 'auth'
```

It's crucial to ensure that all necessary imports are correctly specified to prevent issues related to undefined constants or variables.


<!-- This is an auto-generated comment by PullProofAI -->